tabled_derive/src/lib.rs

```diff
@@ -477,6 +531,17 @@ fn use_function_no_args(function: &str) -> TokenStream {
     }
 }
 
+fn use_format(args: &TokenStream, custom_format: &str) -> TokenStream {
+    quote! { format!(#custom_format, #args) }
+}
+
+fn use_format_no_args(custom_format: &str, field: &TokenStream) -> TokenStream {
+    if custom_format.contains("{}") {
+        return quote! { format!(#custom_format, #field) };
```

Comment:
Is this always will pass a targeted field as first argument to `format!` while ignoring other arguments, in case someone "forgot" `{}` the braces?

I guess so.

I very much believe could be triggered by the test I've described.

__________

See the above comment.
I consider it can be removed if that will be taken in mind.

Reply:
It's removed.